tests/test_treeherder_menu.py

```diff
@@ -0,0 +1,26 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.perfherder import PerfherderPage
+from pages.treeherder import TreeherderPage
+
+
+class TestTreeherderHeader:
```

Comment:
I've stopped using test classes as they don't really add anything unless there's more than one in a file. I would suggest removing this line, unindenting the rest of the file, and removing `self, ` from the test method signatures.